src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,51 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+interface CardProps {
+  movie: Movie;
+}
+
+export const MovieCard: React.FC<CardProps> = ({ movie }) => (
+  <div className="card" data-cy="Movie">
```

Comment:
make a destructuring from movie
```suggestion
export const MovieCard: React.FC<CardProps> = ({ movie }) => {
  const { ... } = movie;
  
  return (
    <div className="card" data-cy="Movie">
```